MAINTAINERS.md

```diff
@@ -0,0 +1,9 @@
+The current Maintainers Group for the [TODO: Projectname] Project consists of:
+
+| Name | Employer | Responsibilities |
+| ---- | -------- | ---------------- |
+|      |          |                  |
+
```

Comment:
Amine Hilaly / AWS / Maintainer

Reply:
Bharanidharan S/ Google/Maintainer

Reply:
Walter F/ Google/Maintainer

Reply:
I think we agreed yesterday to add the "big list" i.e. the 2 or 3 people from each cloud, and then they will self-direct into OWNERs or leads as appropriate.